.github/CODEOWNERS

```diff
@@ -1336,6 +1339,12 @@ x-pack/test/**/deployment_agnostic/ @elastic/appex-qa #temporarily to monitor te
 /x-pack/test_serverless/**/test_suites/common/saved_objects_management/ @elastic/kibana-core
 /x-pack/test_serverless/api_integration/test_suites/common/core/ @elastic/kibana-core
 /x-pack/test_serverless/api_integration/test_suites/**/telemetry/ @elastic/kibana-core
+/x-pack/test/functional/es_archives/cases/migrations/8.8.0 @elastic/kibana-core
+/x-pack/test/functional/es_archives/cases/migrations/7.11.1 @elastic/kibana-core
+/x-pack/test/functional/es_archives/cases/migrations/7.13_user_actions @elastic/kibana-core
+/x-pack/test/functional/es_archives/cases/migrations/7.13.2 @elastic/kibana-core
+/x-pack/test/functional/es_archives/cases/migrations/7.16.0_space @elastic/kibana-core
```

Comment:
Did core create these archives? Perhaps @gsoldevila would know having updated sets of these recently

Reply:
@jloleysens thanks for chiming in as it's rather confusing :)

Reply:
++ looking at the history of https://github.com/elastic/kibana/tree/main/x-pack/test/functional/es_archives/cases/migrations, most of the commiters seem to be from @elastic/response-ops (except this one time Gerard committed to split the Kibana indices).

Reply:
https://github.com/elastic/kibana/pull/194411/commits/d06425759c6868ad279e21222b39bffaed8a2ebe